src/PerformanceTests/Common/Scenarios/BaseRunner.cs

```diff
@@ -48,20 +56,27 @@ public virtual void Execute(Permutation permutation, string endpointName)
         finally
         {
 #if Version5
-            using(EndpointInstance){}
+            using (EndpointInstance) { }
 #else
             EndpointInstance.Stop().GetAwaiter().GetResult();
 #endif
         }
     }
 
+    void ThrowIfPermutationNotAllowed()
```

Comment:
Because running a Saga with InMemory persistence and restarting the endpoint, makes everything disappear and the test worthless. I thought of not throwing an exception, but just ignoring the test. But this was something I wanted to ask, before merging.